lib/brcobranca/retorno/cnab240/sicoob.rb

```diff
@@ -0,0 +1,94 @@
+# -*- encoding: utf-8 -*-
+
+require 'parseline'
+
+module Brcobranca
+  module Retorno
+    module Cnab240
+      class Sicoob < Brcobranca::Retorno::Cnab240::Base
+        # Regex para remoção de headers e trailers além de registros diferentes de T ou U
+        REGEX_DE_EXCLUSAO_DE_REGISTROS_NAO_T_OU_U = /^((?!^.{7}3.{5}[T|U].*$).)*$/
+
+        def self.load_lines(file, options = {})
+          default_options = { except: REGEX_DE_EXCLUSAO_DE_REGISTROS_NAO_T_OU_U }
+          options = default_options.merge!(options)
+
+          Line.load_lines(file, options).each_slice(2).reduce([]) do |retornos, cnab_lines|
+            retornos << generate_retorno_based_on_cnab_lines(cnab_lines)
+          end
+        end
+
+        def self.generate_retorno_based_on_cnab_lines(cnab_lines)
+          retorno = new
+          cnab_lines.each do |line|
+            if line.tipo_registro == 'T'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.